### ISSUE TITLE
Bilioduodenal fistula

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10053]
**Preferred term label:**
Bilioduodenal fistula

**Synonyms**

**Definition (free text, please give PubMed ID)**
Connection between the gallbladder and the duodenum

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Bouveret syndrome (bilioduodenal fistula)

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000453
- label: Bilioduodenal fistula
- parents: [Abnormal gallbladder morphology (HP:0012437)](https://hpo.jax.org/app/browse/term/HP:0012437)
- pmid: PMID:28613489
- def: An abnormal connection between the gallbladder and the duodenum.
- comment: Following an attack of acute cholecystitis, the resulting inflammation and adhesion of the gallbladder to GI, together with mechanical pressure applied by gallstones on the gallbladder itself and bowel wall, may result in an ischemic tear of the apposed gallbladder and enteric wall; this mechanism potentiates a fistula between the gallbladder and bowel where the gallstone could pass. Bouveret syndrome is the most infrequent variant of gallstone ileus. It results from a gallstone migrating through a bilioenteric fistula and causing obstruction at the gastric outlet and occasionally in also other portions of the intestine.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10053
- synonyms: Bouveret syndrome(NONE)
